src/bubo.cc

```diff
@@ -55,8 +55,19 @@ NAN_METHOD(Bubo::Initialize)
         return;
     }
 
-    Local<Value> ignoredVal = Nan::Get(opts, ignoredAttributes).ToLocalChecked();
-    Local<Object> ignored = Nan::To<Object>(ignoredVal).ToLocalChecked();
+    Local<Value> ignored_value = Nan::Get(opts, ignoredAttributes).ToLocalChecked();
+    Local<Object> ignored_array = Nan::To<Object>(ignored_value).ToLocalChecked();
```

Comment:
This is weird but Nan doesn't have `To<Array>` for some reason and I didn't see how to do it. (https://github.com/nodejs/nan/blob/master/doc/converters.md#api_nan_to)